.github/workflows/integration.yaml

```
@@ -9,3 +9,77 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: fernandrone/linelint@0.0.6
+
+  labeling-languages:
```

보통 작업 이름은 `build`, `test`, `lint` 처럼 동사형으로 짓는 게 GitHub Actions에서 좀 더 일반적인 관행인 것 같습니다. 다음과 같이 변경하면 어떨까요?

```suggestion
  label-lang:
```

---

네 깔끔하네요! 변경하였습니다~ 이제 머지도 하겠습니다!